internal/auth/oidc/rewrapping_test.go

```diff
@@ -25,10 +25,10 @@ func TestRewrap_authMethodRewrapFn(t *testing.T) {
 		conn, mock := db.TestSetupWithMock(t)
 		wrapper := db.TestWrapper(t)
 		mock.ExpectQuery(
-			`SELECT \* FROM "kms_schema_version" WHERE 1=1 ORDER BY "kms_schema_version"\."version" LIMIT 1`,
+			`SELECT \* FROM "kms_schema_version" WHERE 1=1 ORDER BY "kms_schema_version"\."version" LIMIT \$1`,
```

Comment:
Why was this change necessary?

Reply:
I believe this was the upstream change: https://github.com/go-gorm/postgres/commit/dc711bd7c69394788e6532ad9c89f5bdcb12b007

Reply:
Thanks!